### ISSUE TITLE
Clarify requirements for setting displayTransformability

### ISSUE THREAD

**Issue by mattgarrish** [w3c/a11y-discov-vocab#85]
The current definition is a bit too generic. It would probably help to beef it up by adding specific properties that must be controllable, and the issues that can limit control (e.g., using `style` attributes or `!important` rules).

**Comment by mattgarrish** [comment]
This should have been closed with https://github.com/w3c/a11y-discov-vocab/pull/86
